<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(194, 158, 255);
color: rgb(160, 255, 101);</string>
    </property>
    <property name="text">
     <string>Product Management</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>351</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Products:</string>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>311</width>
       <height>261</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>309</width>
        <height>259</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutButton"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>391</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Product Details:</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Price:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Discount:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditID">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditName">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPrice">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDiscount">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonOfficial">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Official</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonNotOfficial">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>250</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Official</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSave">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>290</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemove">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>761</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>More Functions:</string>
    </property>
    <widget class="QPushButton" name="pushButtonSearchName">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>40</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Search Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFilterPrice">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>90</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter Price</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditEnterName">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter Price:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditEnterPrice">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
